config/occo.yml

```diff
@@ -28,3 +28,8 @@ entries:
     
 - prefix: /
```

Comment:
Entries are processed in order. This `prefix: /` entry matches everything, so your new entry is never checked. Please move your new entry before this `prefix: /` entry.